apps/rush/src/actions/GenerateAction.ts

```diff
@@ -59,6 +67,22 @@ export default class GenerateAction extends BaseRushAction {
 
     const installManager: InstallManager = new InstallManager(this.rushConfiguration);
 
+    const shrinkwrapFile: ShrinkwrapFile | undefined
+        = ShrinkwrapFile.loadFromFile(this.rushConfiguration.committedShrinkwrapFilename);
+
+    if (shrinkwrapFile
+      && !this._forceParameter.value
+      && installManager.createTempModulesAndCheckShrinkwrap(shrinkwrapFile)) {
+      console.log();
+      console.log(colors.yellow('"rush generate" is unnecessary, since all' +
+        ' direct dependencies already exist in the shrinkwrap file.'));
+      console.log(colors.yellow('Did you mean to run "rush install"?'));
+      console.log();
+      console.log('You can use the \'--force\' flag to bypass this check and forcibly' +
```

Comment:
> \' [](start = 34, length = 3)

In English, we use apostrophes for contractions and possessives.  For quotations, use "